cookbook/core/containerization/multi_images.py

```diff
@@ -46,13 +46,121 @@
 
     It is the responsibility of the user to push a container image that matches the new name described.
 
+If you wish to build and push your Docker image to Dockerhub through your account, follow the below steps:
```

Comment:
@SmritiSatyanV, should the step-by-step guide elaborate on pushing an image to ghcr?

Reply:
Or maybe provide a link to [this](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)